client/app/components/pages/manuscripts.component.ts

```diff
@@ -7,6 +7,7 @@ import { TasksService } from '../../services/tasks.service'
 import { Task } from '../../models/Task';
```

Comment:
looking good! , change the button to be of the 'Assign' button to be with the same style as the others